w2c2/w2c2_base.h

```diff
@@ -185,6 +190,12 @@ typedef double F64;
 #define UNUSED
 #endif
 
+#if defined(_MSC_VER) && _MSC_VER <= 1000
+#define LLONG_MIN (-0x7fffffffffffffffi64-1)
```

Comment:
Given that only the suffixes differ, maybe instead of redefining all constants, we can just define and use a macro like `LL(x)` which adds the appropriate suffix?

Reply:
Nice! Thank you for the refactor 👍